src/main/java/org/sonarsource/scanner/maven/SonarQubeMojo.java

```diff
@@ -57,7 +57,7 @@
 
 public class SonarQubeMojo extends AbstractMojo {
 
-  @Parameter(defaultValue = "${session}", readonly = true)
+  @Parameter(defaultValue = "${session}", readonly = true, required = true)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder annotation arguments to match the order of declaration. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS3340)